src/docs/BUILD.md

```diff
@@ -37,6 +37,37 @@ The preferred approach is to build both OpenVINO and OpenVINO GenAI from sources
 
 ## Build Instructions
 
+### Build OpenVINO GenAI as OpenVINO Extra Module
+
+OpenVINO GenAI can be built as an extra module during the OpenVINO build process. This method simplifies the build process by integrating OpenVINO GenAI directly into the OpenVINO build.
+
+1. Clone OpenVINO repository:
+    > **NOTE**: The path to the OpenVINO repository is referred as `<OPENVINO_REPO_PATH>` throughout the document
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino.git
+    ```
+2. Clone OpenVINO GenAI repository:
+    > **NOTE**: The path to the OpenVINO GenAI repository is referred as `<GENAI_REPO_PATH>` throughout the document
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino_genai.git
+    ```
+3. Create a build directory:
+    > **NOTE**: The path to the OpenVINO build directory is referred as `<OPENVINO_BUILD_PATH>` throughout the document
+    ```sh
+    mkdir <OPENVINO_BUILD_PATH>
```

Comment:
You crate that folder. You can be specific here. You don't need to template it trying to guess how someone could have named it. Replace with just `build`. Same for `<GENAI_REPO_PATH>` and `<OPENVINO_REPO_PATH>`. You know how the folder is going to be named because you provided the exact command for cloning..

Reply:
Fixed